client/src/api/api.js

```diff
@@ -248,12 +249,28 @@ export const uploadAudioFile = async (
 }
 
 /* Audio Endpoints */
+export const downloadAudioFileToExpo = async (vocabId, uri) => {
```

Comment:
Correct me if I'm wrong but based on my understanding of this function it is really more "updating" than "downloading", so a name like `updateAudioFileURI` might be more suitable?

Reply:
Ok maybe do the same with the image function as well!